pravega/pravega-samples
# Issue-245: Update the Flink samples with the new Table API

Signed-off-by: Guangfeng Xu <Guangfeng.Xu@dell.com>

**Change log description**

1. Update the Flink version to 1.10
2. Update the Flink connector version to the latest snapshot
3. Remove the dependency of the package org.joda.time
4. Change TypeInformation to DataType
5. Using the table descriptor way to create a TableSource instead of FlinkPravegaJsonTableSource

**Purpose of the change**
Fixes #245 

**What the code does**

1. Update the configuration of the project
2. Using the table descriptor way to create a TableSource

**How to verify it**

1. `./gradlew clean installDist` should pass
2. Open a pravega standalone cluster
3. `cd $SAMPLES-HOME/scenarios/pravega-flink-connector-sql-samples/build/install/pravega-flink-connector-sql-samples`
4. `bin/tableapi-samples --runApp Prepare` should insert data correctly to pravega
5. `bin/tableapi-samples --runApp PopularDestinationQuery`, `bin/tableapi-samples --runApp PopularTaxiVendor` and `bin/tableapi-samples --runApp MaxTravellers` should show correct results
